### ISSUE TITLE
Enlarging the size of the User Interface does not enlarge the top level Menu bar

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#31758]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

The ability to change appearance settings for Low vision users are a big improvement – the larger text feature is great but once applied, it doesn’t affect the top menu ribbon (File, Edit, View etc), which remains very small.

Steps:
1. From Edit → Preferences, select Appearance.
2. Change the font size from 12pt to, say, 24pt.
3. OK the dialog.

Expected behaviour: all user interface elements should enlarge.
Actual behaviour: the menu bar (File, Edit ...) does not enlarge and remains small.

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.5.2

### Regression

I was unable to check

### Operating system

iOS: Sequoia 15.6

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by jmwmusic** [comment]
Sorry, closing as I realised this is a duplicate of [16179](https://github.com/musescore/MuseScore/issues/16179)
